docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
Do we actually need it? I think a simple Docker command to start Postgres locally would be simpler.

If you decide to keep it, could you please fix it, so that it "just works" for the local setup. I get the following error:
```
$ docker compose up -d
network ingress declared as external, but could not be found
```

Reply:
I removed the external `ingress` network and added docs to explain that one must create the external volume in order to persist data between uses of this service.